portfolio/public/js/index.js

```diff
@@ -0,0 +1,53 @@
+var http = require('http');
+var fs = require('fs');
+var path = require('path');
+// let pathName = __dirname + "/"
+
+var server = http.createServer(requestHandler);
+
+var publicPath = path.join(__dirname, '../..');
+var publicCss =  path.join(__dirname, '../css');
+var  mediaPath = path.join(__dirname, "..")
+console.log(publicPath);
+console.log(publicCss);
+console.log(mediaPath)
+function requestHandler(req, res) {
+    console.log(req.url);
+    
+  // handle all html file together
+  if(req.url === '/') {
+    // set appropriate headers
+    
+    res.setHeader('Content-Type', 'text/html')
+    fs.createReadStream(publicPath+"/templates/index.html").pipe(res);
+    // read file and send chunked data in response
+    // fs.createReadStream(file_path).pipe(res);
+    // for all css files
+  } else if(req.url === '/portfolio'){
+    res.setHeader('Content-Type', 'text/html')
+    fs.createReadStream(publicPath+"/templates/portfolio.html").pipe(res);
+  }
+  else if(req.url.includes('css')) {
```

Comment:
what if the requested image contains some name including css like css1.jpg or something which includes css. Instead check for extensions to better determine which file is requested as you have done for the images.